requirements-freeze.txt

```diff
@@ -2,18 +2,17 @@ Babel==2.3.4
 Beaker==1.9.0
 bleach==3.1.4
 boto==2.49.0
-boto3==1.17.5
-botocore==1.20.5
+boto3==1.17.9
+botocore==1.20.9
 certifi==2020.12.5
 chardet==3.0.4
 -e git+https://github.com/ckan/ckan.git@c04b6d64aadc7a647b8896279002626ca3be0a56#egg=ckan
--e git+https://github.com/GSA/inventory-app.git@4117e40dd38ded07fc458d13e0d7ce330e90c50b#egg=ckanext_datagov_inventory&subdirectory=ckanext-datagov_inventory
```

Comment:
I don't believe this should be removed, I thought @adborden fixed this in main...